plugins/inputs/bigbluebutton/bigbluebutton.go

```diff
@@ -0,0 +1,217 @@
+package bigbluebutton
+
+import (
+	"crypto/sha1"
+	"encoding/xml"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/influxdata/telegraf"
+	"github.com/influxdata/telegraf/plugins/common/proxy"
+	"github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/inputs"
+)
+
+type BigBlueButton struct {
+	URL              string `toml:"url"`
+	PathPrefix       string `toml:"path_prefix"`
+	SecretKey        string `toml:"secret_key"`
+	getMeetingsURL   string
+	getRecordingsURL string
+
+	tls.ClientConfig
+	proxy.HTTPProxy
+	client *http.Client
+}
+
+var defaultPathPrefix = "/bigbluebutton"
+
+var bbbConfig = `
```

Comment:
Can you do me a favor and name it `sampleConfig`. That's the name used in other plugins and even though it's not a must it's easier if you search for it... :-)

Reply:
Resolved.